### ISSUE TITLE
(After April 19) Test redirects that were merged into 17.6 release

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#5144]
### Summary

**What we are after:**
We are getting off transition

### Related issues

- #5111 

### Completion criteria

- [x] Redirects are working
- [x]  For transition redirects, www or Wagtail redirects are put in as needed  
- [x] [Spreadsheet](https://docs.google.com/spreadsheets/d/1vzN-wZlS8K8ZyfExSV3hwXDenDEN9KyyAUJFTI6OjKU/edit?usp=sharing) is updated
- [x]  Pages or files that can be taken down are added to take-down ticket (or just taken down if they were in Wagtail)

### Future work

-Take down ticket for front-end - #5174

**Comment by dorothyyeager** [comment]
Most redirects are working. There's one we apparently missed implementing and one that I need to investigate to see why it's not working.

**Comment by dorothyyeager** [comment]
After reviewing the spreadsheet and the transition redirect code, there's actually five redirects that need to be added. (That's why they aren't working, they weren't done to begin with.)

**Comment by dorothyyeager** [comment]
I've put in a PR for the missing redirects. Once that is merged, this ticket can be closed. We'll test them along with other redirects put in during innovation sprint.

**Comment by dorothyyeager** [comment]
Closing because related PR has been merged. Testing ticket in #5190
